### ISSUE TITLE
`aspect` not work after choose another image

### ISSUE THREAD

**Issue by nanxiaobei** [DominicTobias/react-image-crop#223]
If set `crop.aspect = 100 / 100`, 
Choose first image, crop area is **Square**.
Choose another image, then crop area became **Rectangle**.

https://codesandbox.io/s/72py4jlll6

That's confusing.

**Comment by nanxiaobei** [comment]
@DominicTobias 
Thanks for your replay.
Do you mean calling `this.setState({ crop: { ...crop, width: null } })` in  `onImageLoaded`?
But `this.setState` is asynchronous, `onChange` still got "old" crop.

**Comment by nanxiaobei** [comment]
In this demo: https://codesandbox.io/s/qln4or8qqq,
With `react-image-crop@4.0.1` version, the crop keeps ratio in two images, works fine.
But if upgrade to `react-image-crop@6.0.8`, the crop breaks between two images.

**Comment by nanxiaobei** [comment]
After test, I found the last working version is `6.0.2`.

**Comment by DominicTobias** [comment]
Thanks for this, there was indeed a regression which is fixed in 6.0.9. See updated sandbox - https://codesandbox.io/s/72py4jlll6

Note the check in onImageLoaded to make the library make a new crop for images after the first one:

```js
// Make the library regenerate aspect crops if loading new images.
const { crop } = this.state;
if (crop.aspect && crop.height && crop.width) {
  this.setState({
    crop: { ...crop, height: null },
  });
}
```

**Comment by nanxiaobei** [comment]
Really appreciate your quick response.

**Comment by DominicTobias** [comment]
I have tried to automatically fix this again in 6.0.10, you shouldn't need any special login in `onImageLoaded` now - please give it a try if you're still using the package

**Comment by nanxiaobei** [comment]
Hi, I've tried that, works like a charm. 😝
